data/json/monsters/insect_spider.json

```
@@ -1857,5 +1857,347 @@
     "death_function": [ "NORMAL" ],
     "special_attacks": [ { "type": "leap", "cooldown": 4, "max_range": 4, "allow_no_target": true }, [ "EAT_CROP", 120 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "LARVA", "STUMBLES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_aphid",
+    "type": "MONSTER",
+    "name": { "str": "giant aphid", "str_pl": "giant aphids" },
```

Plural name isn't needed here as game doesn't use it.
```suggestion
    "name": { "str": "giant aphid" },
```